Sources/SwiftSyntaxBuilder/Declarations/Buildables/Variable.swift

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+public protocol VariableMutability {
+  static var token: TokenSyntax { get }
+}
+
+public enum VariableLetMutability: VariableMutability {
+  public static let token = Tokens.let
+}
+
+public enum VariableVarMutability: VariableMutability {
+  public static let token = Tokens.var
+}
+
+public typealias Let = Variable<VariableLetMutability>
+public typealias Var = Variable<VariableVarMutability>
+
+public struct Variable<Mutability: VariableMutability>: DeclBuildable {
```

Comment:
Using generic types here for mutability seems to me an overkill. Can we put `let` or `var` keyword as part of the initializer?

Reply:
Do you mean it should look like `Variable(.let, "something")`? I think it's more natural if we can just write `Let("something")` and `Var("something")`, and afaik we need separate types (or a generic type) for this way.